### PR TITLE
feat(forms): replace our password advice with responsive message

### LINKED ISSUE #5750: Replace our password advice with responsive message
Re-enable the password strength filters and replace "A strong, unique password will keep your Firefox data safe from intruders." with:
password-tips-empty

### PR DESCRIPTION
**Pull request by vladikoff**
Fixes #5750

### CODE REVIEW

**Comment by vladikoff** [comment]
@ryanfeeley please review: https://featurebox.dev.lcip.org/signup

**Comment by ryanfeeley** [comment]
Getting there! No need for blue text in balloon. Point balloon to currently focused password field only (e.g. on **Password** when focused there, on **Repeat password** when focused there)

**Comment by vladikoff** [comment]
TODO: 

- [ ] No need for blue text in balloon
- [ ]  Point balloon to currently focused password field only 
- [ ] Get latest copy fixes from @ryanfeeley

**Comment by ryanfeeley** [comment]
Desktop size version (Grey 90, not bold, headings):

**Don’t reuse your email password**
If someone knows your email password, they can approve your Firefox sign-in on a new device and access your data. 

**Remember your password**
If you need to reset your password, you will lose any Sync data that is not on one of your devices.

Mobile size version due later today…

File: app/scripts/templates/sign_up_password.mustache [hunk 1]
```diff
@@ -21,7 +21,7 @@
 
       <div class="input-row password-row">
         <input id="password" type="password" class="password check-password tooltip-below" placeholder="{{#t}}Password{{/t}}" value="{{ password }}" pattern=".{8,}" required autofocus data-form-prefill="password" data-synchronize-show="true" />
-        <div class="input-help input-help-focused input-help-signup">{{#t}}A strong, unique password will keep your Firefox data safe from intruders.{{/t}}</div>
+        <div class="input-help input-help-focused input-help-signup">{{#t}}Tips: Keep your password safe, and don’t reuse your email password.{{/t}}</div>
```
Comment (vladikoff) [inline app/scripts/templates/sign_up_password.mustache]: Found this: https://github.com/mozilla/fxa-content-server/issues/5947 

will need a follow up PR with more stuff to add that in

**Comment by vladikoff** [comment]
@vbudhram r? One thing is missing is the `Tips:` string from @ryanfeeley but should be good to go otherwise.

**Comment by vladikoff** [comment]
@vbudhram 2 things to test:

1. /signup view should have the new stuff
2. /signup_password (via http://127.0.0.1:3030/?service=sync&context=fx_firstrun_v2&action=email ) should be no bubble for now. Will be added with https://github.com/mozilla/fxa-content-server/issues/5947

File: app/scripts/views/mixins/password-mixin.js [hunk 2]
```diff
@@ -228,11 +229,20 @@ define(function (require, exports, module) {
...
      * password field is focused.
+     *
+     * This also positions the password helper balloon on the correct password field.
+     * @param {Object} event Focus event
      */
-    highlightSignupPasswordHelper () {
+    highlightSignupPasswordHelper (event) {
       this.$(SELECTOR_SIGNUP_PASSWORD_HELPER).addClass('highlight');
+      // the code below moves the signup password helper between the two password fields
+      if (event && this.$(event.target).is('#password')) {
```
Comment (vbudhram) [inline app/scripts/views/mixins/password-mixin.js]: If the passwords don't match, the balloon does not disappear when tabbing to other age field. Seems like it should?

screen shot 2018-03-02 at 11 22 33 am

Comment (vladikoff) [reply app/scripts/views/mixins/password-mixin.js]: > balloon does not disappear when tabbing to age field

@ryanfeeley should it?

**Comment by ryanfeeley** [comment]
@vbudhram Thanks for the eye. Correct. Only a focused password field should ever have the balloon. 

Also I think I spot faux bold there. Make sure font weight is normal even for <b>.

**Comment by ryanfeeley** [comment]
@vladikoff @vbudhram

Mobile copy!

> Don’t lose this password. If you need to reset it, you may lose Sync data.

**Comment by vladikoff** [comment]
@ryanfeeley thank you!

**Pull request merged**